qa/crontab/teuthology-cronjobs

```diff
@@ -138,6 +141,12 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 ###### on smithi
 
 
-# 23 14 * * 6 CEPH_BRANCH=main; MACHINE_NAME=smithi; SUITE_NAME=upgrade:pacific-x; KERNEL=distro; /home/teuthology/bin/cron_wrapper /home/teuthology/bin/schedule_subset.sh 10 $CEPH_BRANCH $MACHINE_NAME $SUITE_NAME $CEPH_QA_EMAIL $KERNEL -p 70 --force-priority
+23 14 * * 6 CEPH_BRANCH=reef; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 100 -m $MACHINE_NAME -s upgrade:pacific-x -k distro -e $CEPH_QA_EMAIL -p 70 --force-priority 
+
+23 14 * * 6 CEPH_BRANCH=reef; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 100 -m $MACHINE_NAME -s upgrade:quincy-x -k distro -e $CEPH_QA_EMAIL -p 70 --force-priority
+
+20 01 * * 4 CEPH_BRANCH=reef; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 100 -m $MACHINE_NAME -s upgrade-clients/client-upgrade-octopus-reef -k distro -e $CEPH_QA_EMAIL --suite-branch octopus --suite-repo https://github.com/ceph/ceph.git --ceph-repo https://github.com/ceph/ceph.git -p 50
```

Comment:
this does not work
And I don't know why?

 
teuthology@teuthology:~$ CEPH_BRANCH=reef; MACHINE_NAME=smithi;teuthology-suite -v -c $CEPH_BRANCH  -n 100 -m $MACHINE_NAME -s upgrade-clients/client-upgrade-octopus-reef -k distro -e $CEPH_QA_EMAIL --suite-branch octopus --suite-repo https://github.com/ceph/ceph.git --ceph-repo https://github.com/ceph/ceph.git -p 50
2023-03-16 16:50:52,117.117 INFO:teuthology.suite:Using random seed=3475
2023-03-16 16:50:52,118.118 INFO:teuthology.suite.run:kernel sha1: distro
2023-03-16 16:50:52,471.471 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph.git reef -> 71abbe2bb8c12479781f0ae91c13320099820d9d
2023-03-16 16:50:52,472.472 INFO:teuthology.suite.run:ceph sha1: 71abbe2bb8c12479781f0ae91c13320099820d9d
2023-03-16 16:50:52,472.472 INFO:teuthology.suite.run:skipping ceph package verification
2023-03-16 16:50:52,828.828 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph.git octopus -> 41ad4cd0bd429a0054881c54d106a1090c55870d
2023-03-16 16:50:53,139.139 DEBUG:teuthology.repo_utils:git ls-remote https://github.com/ceph/ceph.git octopus -> 41ad4cd0bd429a0054881c54d106a1090c55870d
2023-03-16 16:50:53,140.140 INFO:teuthology.suite.run:ceph branch: octopus 41ad4cd0bd429a0054881c54d106a1090c55870d
2023-03-16 16:50:53,141.141 DEBUG:teuthology.repo_utils:Setting repo remote to https://github.com/ceph/ceph.git
2023-03-16 16:50:53,148.148 ERROR:teuthology.repo_utils:b'fatal: not a git repository (or any of the parent directories): .git\n'
2023-03-16 16:50:53,148.148 ERROR:teuthology.repo_utils:Git error encountered; retrying
Traceback (most recent call last):
  File "/home/teuthology/src/git.ceph.com_git_teuthology_main/teuthology/repo_utils.py", line 363, in fetch_repo
    enforce_repo_state(url, dest_path, branch, commit)
  File "/home/teuthology/src/git.ceph.com_git_teuthology_main/teuthology/repo_utils.py", line 117, in enforce_repo_state
    set_remote(dest_path, repo_url)
  File "/home/teuthology/src/git.ceph.com_git_teuthology_main/teuthology/repo_utils.py", line 243, in set_remote
    raise GitError("git remote set-url failed!")
teuthology.exceptions.GitError: git remote set-url failed!

2023-03-16 16:51:03,157.157 DEBUG:teuthology.repo_utils:Setting repo remote to https://github.com/ceph/ceph.git
2023-03-16 16:51:03,163.163 ERROR:teuthology.repo_utils:b'fatal: not a git repository (or any of the parent directories): .git\n'
2023-03-16 16:51:03,164.164 ERROR:teuthology.repo_utils:Git error encountered; retrying
Traceback (most recent call last):
  File "/home/teuthology/src/git.ceph.com_git_teuthology_main/teuthology/repo_utils.py", line 363, in fetch_repo
    enforce_repo_state(url, dest_path, branch, commit)
  File "/home/teuthology/src/git.ceph.com_git_teuthology_main/teuthology/repo_utils.py", line 117, in enforce_repo_state
    set_remote(dest_path, repo_url)
  File "/home/teuthology/src/git.ceph.com_git_teuthology_main/teuthology/repo_utils.py", line 243, in set_remote
    raise GitError("git remote set-url failed!")
teuthology.exceptions.GitError: git remote set-url failed!


@jdurgin maybe because we forced-merged? Does something have to do with the `octopus` branch?